hybrid_content_manager/hybridmanager.py

```diff
@@ -186,13 +190,29 @@ def root_manager(self):
     def _extra_root_dirs(self):
         return [base_directory_model(path) for path in self.managers if path]
 
+    def _validate_path(self, prefix, path):
+        validator = self.path_validator.get(prefix, lambda path: True)
+
+        path_is_valid = validator(path)
+
+        # Ensure return type is bool
+        if type(path_is_valid) is not bool:
+            print('Path validator does not return a boolean.')
+
+        if not path_is_valid:
+            raise HTTPError(
+                401,
+                "The provided path_validator for the prefix '{prefix}' has flagged the path '{path}' as invalid."
```

Comment:
`"The path '{path}' is not valid for the prefix '{prefix}'`

Reply:
Also raise a `400` error https://developer.mozilla.org/en-US/docs/Web/HTTP/Status